co-simulation/bundle/src/assembly/resources/fed/ns3/ns3_installer.sh

```diff
@@ -60,17 +60,18 @@ ns3_version_affix="ns-allinone-$ns3_version"
 ns3_version_affix_unified="ns-allinone" #deprecated, not used momentarily
 ns3_short_affix="ns-$ns3_version"
 ns3_short_affix_unified="ns3"
-ns3_deploy_folder="ns3-deployed"  #name to be used when ns3 is deployed (i.e. keep only binaries)
-working_directory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ns3_deploy_folder="ns3-deployed" #name to be used when ns3 is deployed (i.e. keep only binaries)
+working_directory="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 federate_path="bin/fed/ns3"
 ns3_installation_path=${working_directory}
-ns3_simulator_folder="${ns3_installation_path}/$ns3_version_affix/$ns3_short_affix" #due to the ns3 tarball structure
+ns3_allinone_folder="${ns3_installation_path}/$ns3_version_affix"
+ns3_simulator_folder="${ns3_allinone_folder}/$ns3_short_affix" #due to the ns3 tarball structure
 ns3_scratch="${ns3_simulator_folder}/scratch"
-ns3_source="${ns3_simulator_folder}/src"
 
 ####### semi automatic parameters ########
-ns3_federate_url="https://github.com/mosaic-addons/ns3-federate/archive/21.0.zip"
+ns3_federate_url="https://github.com/usdot-fhwa-stol/ns3-federate.git"
 ns3_url="http://www.nsnam.org/release/$ns3_version_affix.tar.bz2"
+ns3_cv2x="https://github.com/usdot-fhwa-stol/ns-3_c-v2x"
```

Comment:
I take it you resolved whatever issue there was with pulling this repo into the build?

Reply:
Yes, the issue has been addressed. The problem with ns3_cv2x I referenced earlier because I modified this [line](https://github.com/usdot-fhwa-stol/ns-3_c-v2x/blob/9f12e55a76cf4cf32e57efb41d06625626256367/waf#L1) in the waf file of ns-3_c-v2x from `#!/usr/bin/env python` to `#!/usr/bin/python3.6`. However, making this change and then pushing it to GitHub disrupted other parts of the waf file. As a result, I've opted to update the build command [here](https://github.com/usdot-fhwa-stol/cdasim/blob/a440aef40f299a38cc5e2e88cbffaf25826cf6dd/co-simulation/bundle/src/assembly/resources/fed/ns3/ns3_installer.sh#L375-L376) to include python version rather than altering the waf file directly.